### ISSUE TITLE
Home Assistant 2024.5

### ISSUE THREAD

**Issue by damacus** [onedr0p/containers#1016]
I'm just going to jot down all the bugs I'm seeing in the container right now, to leave note for us later on 👍

DHCP Error
```
Logger: aiodhcpwatcher
Source: /usr/local/lib/python3.12/site-packages/aiodhcpwatcher/__init__.py:109
First occurred: 10:53:48 (1 occurrences)
Last logged: 10:53:48

Cannot watch for dhcp packets without a functional packet filter: libpcap is not available. Cannot compile filter !
```

Bluetooth device tracker 
```
Logger: homeassistant.components.device_tracker
Source: components/device_tracker/legacy.py:361
integration: Device Tracker (documentation, issues)
First occurred: 10:53:47 (1 occurrences)
Last logged: 10:53:47

Error setting up platform legacy bluetooth_le_tracker
```

Bluetooth related

```
Logger: homeassistant.util.package
Source: util/package.py:123
First occurred: 10:53:26 (3 occurrences)
Last logged: 10:53:47

Unable to install package PyBluez==0.22: error: subprocess-exited-with-error × python setup.py bdist_wheel did not run successfully. │ exit code: 1 ╰─> [21 lines of output] running bdist_wheel running build running build_py creating build creating build/lib.linux-aarch64-cpython-312 creating build/lib.linux-aarch64-cpython-312/bluetooth copying bluetooth/widcomm.py -> build/lib.linux-aarch64-cpython-312/bluetooth copying bluetooth/__init__.py -> build/lib.linux-aarch64-cpython-312/bluetooth copying bluetooth/ble.py -> build/lib.linux-aarch64-cpython-312/bluetooth copying bluetooth/bluez.py -> build/lib.linux-aarch64-cpython-312/bluetooth copying bluetooth/osx.py -> build/lib.linux-aarch64-cpython-312/bluetooth copying bluetooth/msbt.py -> build/lib.linux-aarch64-cpython-312/bluetooth copying bluetooth/btcommon.py -> build/lib.linux-aarch64-cpython-312/bluetooth warning: build_py: byte-compiling is disabled, skipping. running build_ext building 'bluetooth._bluetooth' extension creating build/temp.linux-aarch64-cpython-312 creating build/temp.linux-aarch64-cpython-312/bluez gcc -fno-strict-overflow -Wsign-compare -DNDEBUG -g -O3 -Wall -fPIC -I./port3 -I/usr/local/include/python3.12 -c bluez/btmodule.c -o build/temp.linux-aarch64-cpython-312/bluez/btmodule.o error: command 'gcc' failed: No such file or directory [end of output] note: This error originates from a subprocess, and is likely not a problem with pip. ERROR: Failed building wheel for PyBluez ERROR: Could not build wheels for PyBluez, which is required to install pyproject.toml-based projects
```

**Comment by damacus** [comment]
Oh yeah, and HACS

```
Logger: homeassistant.setup
Source: setup.py:322
First occurred: 10:53:19 (1 occurrences)
Last logged: 10:53:19

Setup failed for custom integration 'hacs': Unable to import component: No module named 'securesystemslib.util'
Traceback (most recent call last):
  File "/usr/local/lib/python3.12/site-packages/homeassistant/setup.py", line 322, in _async_setup_component
    component = await integration.async_get_component()
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/homeassistant/loader.py", line 996, in async_get_component
    self._component_future.result()
  File "/usr/local/lib/python3.12/site-packages/homeassistant/loader.py", line 976, in async_get_component
    comp = await self.hass.async_add_import_executor_job(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/homeassistant/loader.py", line 1036, in _get_component
    ComponentProtocol, importlib.import_module(self.pkg_path)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/homeassistant/util/loop.py", line 144, in protected_loop_func
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/config/custom_components/hacs/__init__.py", line 12, in <module>
    from aiogithubapi import AIOGitHubAPIException, GitHub, GitHubAPI
  File "/usr/local/lib/python3.12/site-packages/aiogithubapi/__init__.py", line 20, in <module>
    from .device import GitHubDeviceAPI
  File "/usr/local/lib/python3.12/site-packages/aiogithubapi/device.py", line 14, in <module>
    from .client import GitHubClient
  File "/usr/local/lib/python3.12/site-packages/aiogithubapi/client.py", line 27, in <module>
    from .legacy.client import AIOGitHubAPIClient as LegacyAIOGitHubAPIClient
  File "/usr/local/lib/python3.12/site-packages/aiogithubapi/legacy/client.py", line 15, in <module>
    from ..helpers import async_call_api
  File "/usr/local/lib/python3.12/site-packages/aiogithubapi/helpers.py", line 8, in <module>
    from sigstore.verify import (
  File "/usr/local/lib/python3.12/site-packages/sigstore/verify/__init__.py", line 54, in <module>
    from sigstore.verify.models import (
  File "/usr/local/lib/python3.12/site-packages/sigstore/verify/models.py", line 30, in <module>
    from sigstore._internal.rekor import RekorClient
  File "/usr/local/lib/python3.12/site-packages/sigstore/_internal/rekor/__init__.py", line 19, in <module>
    from .client import RekorClient
  File "/usr/local/lib/python3.12/site-packages/sigstore/_internal/rekor/client.py", line 35, in <module>
    from sigstore._internal.tuf import TrustUpdater
  File "/usr/local/lib/python3.12/site-packages/sigstore/_internal/tuf.py", line 28, in <module>
    from tuf.ngclient import Updater
  File "/usr/local/lib/python3.12/site-packages/tuf/ngclient/__init__.py", line 8, in <module>
    from tuf.api.metadata import TargetFile
  File "/usr/local/lib/python3.12/site-packages/tuf/api/metadata.py", line 59, in <module>
    from securesystemslib.util import persist_temp_file
ModuleNotFoundError: No module named 'securesystemslib.util'
```

**Comment by onedr0p** [comment]
Maybe this will fix the `Setup failed for custom integration 'hacs': Unable to import component: No module named 'securesystemslib.util'` error?

https://github.com/hacs/integration/pull/3665

**Comment by onedr0p** [comment]
Looks like hass released 2024.5.1, did you try this out?

**Comment by damacus** [comment]
Yep, looks like it's all good again.
